pkg/expect/expect.go

```diff
@@ -122,22 +124,82 @@ func (ep *ExpectProcess) read() {
 func (ep *ExpectProcess) tryReadNextLine(r *bufio.Reader) error {
 	printDebugLines := os.Getenv("EXPECT_DEBUG") != ""
 	l, err := r.ReadString('\n')
-
+	f, err1 := os.Create("logs.txt")
+	if err1 != nil {
+		fmt.Printf("error: %v", err1)
+	}
+	defer f.Close()
 	ep.mu.Lock()
 	defer ep.mu.Unlock()
 
 	if l != "" {
 		if printDebugLines {
-			fmt.Printf("%s (%s) (%d): %s", ep.cmd.Path, ep.cfg.name, ep.cmd.Process.Pid, l)
+			logVal := fmt.Sprintf("%s (%s) (%d): %s", ep.cmd.Path, ep.cfg.name, ep.cmd.Process.Pid, l)
```

Comment:
Please provide an option to configure if logs are written to file. Depending when and where we run the test we might prefer to get logs to stdout